bitrise/template_util.go

```diff
@@ -23,14 +24,25 @@ type TemplateDataModel struct {
 }
 
 func getEnv(key string, envList envmanModels.EnvsJSONListModel) string {
-	if len(envList) > 0 {
+	envfilePath := os.Getenv(envfile.DefaultEnvfilePathEnv)
+	_, err := os.Stat(envfilePath)
+	if errors.Is(err, os.ErrNotExist) {
 		for aKey, value := range envList {
 			if aKey == key {
 				return value
 			}
 		}
+		return os.Getenv(key)
 	}
-	return os.Getenv(key)
+
+	value, err := envfile.GetEnv(key, envList, envfilePath)
+	if err != nil {
+		log.Warnf("Failed to read env from envfile: %s", err)
```

Comment:
In this case shouldn't we do the same as when envfilePath doesn't exist (i.e.: read the from envlist and fallback to os.getenv, just like above)?